### PR TITLE
[android][host] fix Info() failure due to forbidden /proc/stat and /proc/uptime

### PR DESCRIPTION
**Pull request by tmm1**
these are not accessible on Android

### CODE REVIEW

**Comment by tmm1** [comment]
> We could use https://pkg.go.dev/golang.org/x/sys/unix#Sysinfo, maybe as a fallback to a read error.

Good idea. I have implemented this approach on the PR, and tested working with Android 11

**Comment by tmm1** [comment]
@Lomanic wdyt?

**Comment by tmm1** [comment]
Friendly ping

File: internal/common/common_linux.go [hunk 1]
```diff
@@ -68,7 +69,17 @@ func BootTimeWithContext(ctx context.Context) (uint64, error) {
...
+		var info syscall.Sysinfo_t
+		err := syscall.Sysinfo(&info)
+		if err != nil {
+			return 0, err
+		}
+
+		currentTime := time.Now().UnixNano() / int64(time.Second)
+		t := currentTime - int64(info.Uptime)
+		return uint64(t), nil
+	} else if err != nil {
```
Comment (Lomanic) [inline internal/common/common_linux.go:82]: useless `else` when you `return` unconditionally in the preceding `if`

Comment (tmm1) [reply internal/common/common_linux.go:82]:
```suggestion
	}
	if err != nil {
```

**Comment by tmm1** [comment]
Thanks. How this now?

**Comment by tmm1** [comment]
ping @shirou

**Pull request merged**